config.example.toml

```diff
@@ -603,6 +603,13 @@ change-id = 116881
 # desired in distributions, for example.
 #rpath = true
 
+# Indicates whether `rustc` should be stripped of symbols using `-Cstrip=symbols`.
+#strip = false
+
+# Indicates whether `rustc` should be compiled with stack protectors.
+# Valid options are : `none`(default),`basic`,`strong`, or `all`.
+#stack-protector = "none"
```

Comment:
I think we should *at least* mention here that it sounds like this is pretty iffy in terms of usage (https://github.com/rust-lang/rust/issues/114903 has comments that look concerning to me). Can we link the tracking issue and note that this may be buggy?

(I was going to ask whether should this be default to not "none", but it sounds like we're not ready for that yet).

Reply:
Modified according to comments.